lib/status/status.js

```diff
@@ -1,10 +1,47 @@
 const { resolve } = require('path')
 const { mkdir, readFile, writeFile } = require('fs-extra')
+const GitHub = require('github-api')
```

Comment:
Why not use the official Github API client https://github.com/octokit/rest.js ?

Reply:
Thanks for the pointer, I'll look into it. I just searched for "node github api" and took what looked the most sensible option :)

Reply:
Updated to use @octokit/rest